mreg_cli/exceptions.py

```diff
@@ -68,6 +68,36 @@ class DeleteFailure(CliError):
     pass
 
 
+class GetFailure(CliError):
```

Comment:
Inconsistent naming scheme? Is there a semantic difference in how exceptions named `*Failure` and `*Error` are used? If yes, it should be probably documented, if not shouldn't we just use `Error` everywhere?